torch/_inductor/quantized_lowerings.py

```diff
@@ -133,6 +134,21 @@ def int4pack_mm_cpu(
             if use_aten_gemm_kernels()
             else []
         )
+        if use_cpp_gemm_template(
```

Comment:
Do we need to check `use_max_autotune()`?

Reply:
Thanks. Added.